content/en/docs/instrumentation/python/mypy.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Using mypy
+weight: 5
```

Comment:
Should this "note" page be at the end of the Python-section page list?

(If you drop the weight it might migrate to the end of the list.)